docs/users-guide/custom-questions.md

```diff
@@ -97,14 +97,20 @@ Here are the basic types of joins:
 
 **A left outer join example:** If Table A is Orders and Table B is Customers, and assuming you do a join where the `customer_id` column in Orders is equal to the `ID` column in Customers, when you do a left outer join your results will be a full list of all your orders, and each order row will also display the columns of the customer who placed that order. Since a single customer can place many orders, a given customer's information might be repeated many times for different order rows. If there isn't a corresponding customer for a given order, the order's information will be shown, but the customer columns will just be blank for that row.
 
-##### Multiple stages of joins
+### Multiple stages of joins
```

Comment:
"Multiple stages of joins" and "Joining on multiple columns" seem like subsections of "Joining data" to me, rather than being at the same level.